### PR TITLE
Feat/teams api

### PR DESCRIPTION
**Pull request by castrocami**
### Contexto
Reconstruyendo Lakatan. Es este [pitch](https://www.notion.so/platanus/Interfaz-de-equipos-v1-0-1c4cd55247ce4ec5b0d94dad66cc294a), que corresponde a CRUDs de equipos
### Qué se esta haciendo
​Crear Api con power_api para teams y organizations
#### En particular hay que revisar
​

-----------

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   scope path: '/api' do
     api_version(module: 'Api::V1', path: { value: 'v1' }, defaults: { format: 'json' }) do
+      resources :teams, only: [:show, :update, :destroy]
+      resources :organizations do
+        resources :teams, only: [:index, :create]
+      end
```
Comment (aaguilarg) [inline config/routes.rb:7]: por que hay dos rutas para teams?

File: spec/integration/api/v1/organizations_spec.rb [hunk 2]
```diff
@@ -0,0 +1,114 @@
+require 'swagger_helper'
+
+describe 'API V1 Organizations', swagger_doc: 'v1/swagger.json' do
```
Comment (monkeyci) [inline spec/integration/api/v1/organizations_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/EmptyExampleGroup: Empty example group detected.

Comment (monkeyci) [inline spec/integration/api/v1/organizations_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/DescribeClass: The first argument to describe should be the class or module being tested.

File: spec/integration/api/v1/teams_spec.rb [hunk 3]
```diff
@@ -0,0 +1,120 @@
+require 'swagger_helper'
+
+describe 'API V1 Teams', swagger_doc: 'v1/swagger.json' do
```
Comment (monkeyci) [inline spec/integration/api/v1/teams_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/EmptyExampleGroup: Empty example group detected.

Comment (monkeyci) [inline spec/integration/api/v1/teams_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/MultipleMemoizedHelpers: Example group has too many memoized helpers [7/5]

Comment (monkeyci) [inline spec/integration/api/v1/teams_spec.rb:3]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/DescribeClass: The first argument to describe should be the class or module being tested.

File: spec/swagger/v1/definition.rb [hunk 4]
```diff
@@ -6,5 +6,11 @@
   },
   basePath: '/api/v1',
   definitions: {
+    team: TEAM_SCHEMA,
+    teams_collection: TEAMS_COLLECTION_SCHEMA,
+    team_resource: TEAM_RESOURCE_SCHEMA,
+    organization: ORGANIZATION_SCHEMA,
+    organizations_collection: ORGANIZATIONS_COLLECTION_SCHEMA,
+    organization_resource: ORGANIZATION_RESOURCE_SCHEMA,
```
Comment (monkeyci) [inline spec/swagger/v1/definition.rb:14]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.

**Pull request merged**